### ISSUE TITLE
SqlGeography.ToString() produce incorrect value

### ISSUE THREAD

**Issue by MaceWindu** [dotMorten/Microsoft.SqlServer.Types#16]
Thanks for your fixes! Our tests work now except one case below:
```

  Expected: "LINESTRING (-122.36 47.656, -122.343 47.656)"
  But was:  "LINESTRING (47.656 -122.36, 47.656 -122.343)"
```
```cs
[Test, IncludeDataContextSource(ProviderName.SqlServer2008, ProviderName.SqlServer2012, ProviderName.SqlServer2014, TestProvName.SqlAzure)]
		public void Test4(string context)
		{
			using (var conn = new DataConnection(context))
			{
				using (var cmd = conn.CreateCommand())
				{
					cmd.CommandText = "SELECT Cast(geography::STGeomFromText('LINESTRING(-122.360 47.656, -122.343 47.656)', 4326) as geography)";

					Assert.That(cmd.ExecuteScalar().ToString(), Is.EqualTo("LINESTRING (-122.36 47.656, -122.343 47.656)"));
				}
			}
		}
```

**Comment by dotMorten** [comment]
Fixed in https://github.com/dotMorten/Microsoft.SqlServer.Types/commit/16b9c3eb5bdf83817e0200b5d6e86632fb543583

**Comment by dotMorten** [comment]
@MaceWindu Thank you again for the easy reproducers! Makes it really easy to reproduce and fix :-)
I've added all your test cases to the unit test project.

**Comment by MaceWindu** [comment]
Great! Now all our related tests pass!

**Comment by dotMorten** [comment]
Awesome! Thank you again! Your bug rpeorts were awesome and helped me quickly fix some pretty critical issues I can't believe I've overlooked.

I've published v1.1.0 on nuget.org
